README.md

```diff
@@ -1,56 +1,116 @@
+
+# Key features
+* Transparent Scalability
+* Ultra-fast Performance
+* Real-time Operational Analytics
+* Continuous Availability
+* MySQL Compatible
+* Cost Effeciency
+
+See [key features](https://en.oceanbase.com/product/opensource) to learn more details.
+
+# System architecture
 ![image.png](https://cdn.nlark.com/yuque/0/2022/png/25820454/1667369873624-c1707034-471a-4f79-980f-6d1760dac8eb.png)
 
-## Roadmap
+[Learn More](https://en.oceanbase.com/docs/community-observer-en-10000000000829641)
+# Quick start
+
+## How to deploy
+
+### 🔥 Deploy by all-in-one
+
+You can quickly deploy a standalone OceanBase Database to experience with the following commands.
+
+```shell
+# download and install all-in-one package (internet connection is required)
+bash -c "$(curl -s https://obbusiness-private.oss-cn-shanghai.aliyuncs.com/download-center/opensource/oceanbase-all-in-one/installer.sh)"
+source ~/.oceanbase-all-in-one/bin/env.sh
+
+# quickly deploy OceanBase database
+obd demo
+```
+### 🐳 Deploy by docker
 
-![image.png](https://cdn.nlark.com/yuque/0/2022/png/25820454/1667369873613-44957682-76fe-42c2-b4c7-9356ed5b35f0.png)
+1. Pull OceanBase image (optional)
+```shell
+docker pull oceanbase/oceanbase-ce
+```
+2. Start an OceanBase Database instance
+```shell
+# Deploy an instance with the maximum specifications supported by the container.
+docker run -p 2881:2881 --name obstandalone -e MINI_MODE=0 -d oceanbase/oceanbase-ce
+# Or deploy a mini standalone instance.
+docker run -p 2881:2881 --name obstandalone -e MINI_MODE=1 -d oceanbase/oceanbase-ce
+```
+3. Connect to the OceanBase Database instance
+```shell
+docker exec -it obstandalone ob-mysql sys # Connect to the root user of the sys tenant.
+docker exec -it obstandalone ob-mysql root # Connect to the root user of the test tenant.
+docker exec -it obstandalone ob-mysql test # Connect to the test user of the test tenant.
+```
 
-## Case study
+See also [Quick experience](https://en.oceanbase.com/docs/community-observer-en-10000000000829647) or [Quick Start (Simplified Chinese)](https://www.oceanbase.com/docs/common-oceanbase-database-cn-10000000001692850) for more details.
 
-For our success stories, see [Success stories](https://www.oceanbase.com/en/customer/home).
+## How to build
 
-## Contributing
+See [OceanBase Developer Document](https://github.com/oceanbase/oceanbase/wiki/Compile) to learn how to compile and deploy a munually compiled observer.
 
-Your contributions to our code will be highly appreciated. For details about how to contribute to OceanBase, see [Contribute to OceanBase](https://github.com/oceanbase/oceanbase/wiki/Contribute-to-OceanBase).
+# Roadmap
 
-## Licensing
+See [OceanBase Roadmap](https://github.com/orgs/oceanbase/projects).
+# Case study
+
```

Comment:
This is two simple, Maybe we can use 

```
Serving 400+ Customers, Big and Small
OceanBase makes data management and use easier with technology innovation. It has been serving more than 400 customers upgrade their database from different industries, including Financial Services, Telecom, Retail, Internet and more.
```

The link could be https://github.com/oceanbase/oceanbase/issues/1301